src/Microsoft.ML/Runtime/EntryPoints/InMemoryDataView.cs

```diff
@@ -0,0 +1,37 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.ML.Runtime;
+using Microsoft.ML.Runtime.CommandLine;
+using Microsoft.ML.Runtime.Data;
+using Microsoft.ML.Runtime.EntryPoints;
+
+[assembly: LoadableClass(typeof(void), typeof(InMemoryDataView), null, typeof(SignatureEntryPointModule), "InMemoryDataView")]
+namespace Microsoft.ML.Runtime.EntryPoints
+{
+    public class InMemoryDataView
```

Comment:
>InMemoryDataView [](start = 17, length = 16)

So to clarify, all this entrypoint does it turns input to output?  Should we call it as such, something like a data passthrough or something?